Fix frontend unit tests for Node 20 (#4607)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 9.9.0 - xxxx-xx-xx =

> * Dev - Multiple updates to the frontend unit tests in preparation for the Node 20 upgrade

Reviewer: Non-blocking nit: I think we could simplify the description, but I don't feel strongly about it.
```suggestion
* Dev - Update Javascript unit tests for compatibility with Node 20
```
